src/framework/core/js/FluidView.js

```diff
@@ -21,13 +21,13 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
 
     fluid.defaults("fluid.viewComponent", {
         gradeNames: ["fluid.modelComponent"],
-        initFunction: "fluid.initView",
         argumentMap: {
             container: 0,
             options: 1
         },
-        members: { // Used to allow early access to DOM binder via IoC, but to also avoid triggering evaluation of selectors
-            dom: "@expand:fluid.initDomBinder({that}, {that}.options.selectors)"
+        members: {
+            container: "@expand:fluid.containerForViewComponent({that}, {that}.options.container)",
+            dom: "@expand:fluid.initDomBinder({that}, {that}.options.selectors, {that}.container)"
```

Comment:
Do members support priorities for when they resolve? In UIO+ there is a"[content](https://github.com/jobara/gpii-chrome-extension/blob/master/extension/src/content_scripts/contentView.js#L49-L54)" member that uses an expander to trigger a function that uses [that.locate](https://github.com/jobara/gpii-chrome-extension/blob/master/extension/src/content_scripts/contentView.js#L87). At the moment the error "that.locate is not a function" is thrown; likely because the domBinder has yet to be instantiated.

Reply:
They don't support priorities, and it would make a total mess of everyone's definitions if they needed to write such things everywhere. There are two things going on here - firstly, there is a bug in very old definition kept over from the old framework that assigns that.locate in a faulty, side-effectful way that means that the framework can't determine where it is coming from. Secondly, there is the issue we were talking about in the channel the other week, that definitions written in code need to be assisted by proxies so that they can force the framework to evaluate material like that.locate which hasn't yet been visited.
I'm totally amazed that the definition you have in UIO+ ever worked in the old framework and I can't see a clear reason for it, although it might have been something to do with the very odd way that viewComponents were instantiated through custom "init functions".
Firstly I need to fix the bug left in this branch which assigns to that.locate side-effectfully via fluid.initDomBinder, and then after I've done this you will be able to pass {that}.locate explicitly as an argument into your expander in the way we currently do to force evaluation (until we have proxies)

Reply:
@amb26 thanks for the response. +1 for things just working and not having to write priorities. 

I'm assuming things are breaking now because it tries to resolve my member before the "dom" member. Maybe I was just lucky before with the timing. 

I'll hold off on continuing my integration efforts until after the bug for the assignment of that.locate has been addressed.

Reply:
Hi there @jobara - you were "lucky" before because the initialisation of the DOM binder used to be special-cased in the old framework by means of the "init function" initViewComponent - init functions have now been abolished so all material attached to the component now needs to take its chances with the ordinary data-driven workflow - which was broken for that.locate since we still retained the old side-effectful function fluid.initDomBinder from the old framework. I've issued a fix that regularises the initialisation of the DOM binder (which is now defined in a new base grade fluid.baseViewComponent) and rolled a new dev release for you to test at version 3.0.0-dev.20190913T161245Z.3910173d0.FLUID-6148 . fluid.initDomBinder is now abolished and only the more reasonable fluid.createDomBinder remains.

Do give it a try when you can and see if you get get a bit further with UIO+. You will unfortunately need to supply "{that}.locate" as a surplus final argument to your locateInContext functions but at least it will now be possible for you to force the evaluation somehow.
These kinds of issues seem to make it more urgent for use to move to the proxy-wrapped framework described in https://issues.fluidproject.org/browse/FLUID-6372 so we can get rid of these surplus arguments.